### PR TITLE
Visual Studio 2019 build

### PR DESCRIPTION
**Pull request by PiotrFusik**

### CODE REVIEW

File: build_win.bat [hunk 1]
```diff
@@ -6,21 +6,21 @@ IF NOT DEFINED ProgFilesDir SET ProgFilesDir=%ProgramFiles%
...
 
-call "%VS140COMNTOOLS%\vsvars32.bat"
+call "%VS160COMNTOOLS%\vsvars32.bat"
 
 mkdir build_win
 pushd build_win
 IF NOT EXIST CLConform.sln (
    echo "Solution file not found, running Cmake"
-   cmake -G "Visual Studio 14 2015 Win64" ..\.  -DKHRONOS_OFFLINE_COMPILER=<TO_SET> -DCL_LIBCLCXX_DIR=<TO_SET> -DCL_INCLUDE_DIR=<TO_SET> -DCL_LIB_DIR=<TO_SET> -DCMAKE_RUNTIME_OUTPUT_DIRECTORY=. -DOPENCL_LIBRARIES=OpenCL
+   cmake -G "Visual Studio 16 2019" ..\.  -DKHRONOS_OFFLINE_COMPILER="khronos_offline_compiler" -DCL_LIBCLCXX_DIR="cl_libclcxx_dir" -DCL_INCLUDE_DIR="C:/work/igc/vpg-compute-neo/third_party/opencl_headers" -DCL_LIB_DIR="C:/Program Files (x86)/inteloneapi/compiler/2021.1-beta03/windows/lib" -DCMAKE_RUNTIME_OUTPUT_DIRECTORY=. -DOPENCL_LIBRARIES=OpenCL
```
Comment (PiotrFusik) [inline build_win.bat:23]: Beware that here are paths specific to my machine.

Comment (gwawiork) [reply build_win.bat:23]: I will not merge this chage. I am using command from cmd or have my private script

**Review by gwawiork — COMMENTED** [review]
changes in test_workgroup will be merged

File: build_win.bat (hunk 1 above)
Comment (PiotrFusik) [reply build_win.bat:23]: OK

**Pull request closed without merging**